sirbot/core/registry.py

```diff
@@ -53,3 +53,5 @@ def __len__(self):
 
     def __contains__(self, item):
         return item in self._plugins
+
+Registry = RegistrySingleton()
```

Comment:
I'd make this just a normal instance name: `registry`. It's a little jarring to see `Registry['value' ]`